### PR TITLE
ac: add page

### PR DESCRIPTION
**Pull request by vmchale**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines

### CODE REVIEW

File: pages/common/ac.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# ac
+
+> A faster version of cat that can strip ANSI color codes.
+
+- Dump a file's contents:
+
+`ac file`
```
Comment (sbrl) [inline pages/common/ac.md]: Perhaps you could make file `{{path/to/file}}` - The `{{` and `}}` are our way of indicating that a user can alter a parameter.

File: pages/common/ac.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+- Strip color codes from output of some command:
+
+`curl -s wttr.in/LA | ac -s`
```
Comment (sbrl) [inline pages/common/ac.md]: Same here. Perhaps change that url to example.com or something and wrap it in `{{` and `}}`?

File: pages/common/ac.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# ac
+
+> A faster version of cat that can strip ANSI color codes.
```
Comment (agnivade) [inline pages/common/ac.md]: put cat within backticks.

**Comment by php-coder** [comment]
To what package this command belongs to? I have something but it does a different things:
```console
$ man ac | col -b | head
AC(1)								  General Commands Manual							     AC(1)

NAME
       ac -  print statistics about users' connect time

SYNOPSIS
       ac     [ -d | --daily-totals ] [ -y | --print-year ]
	      [ -p | --individual-totals ] [ people ]
	      [ -f | --file filename ] [ -a | --all-days ]
	      [ --complain ] [ --reboots ] [ --supplants ]
$ rpm -qf /usr/bin/ac
psacct-6.6.2-4.fc24.x86_64
```

**Comment by agnivade** [comment]
https://github.com/vmchale/fastcat

**Comment by agnivade** [comment]
Closing this due to inactivity. @vmchale - Please feel free to comment on this thread whenever you are ready to take this forward.

**Pull request closed without merging**